### DISCUSSION TITLE
I do not see NootedRed.kext  file

### DISCUSSION THREAD

**Discussion by kofany** [ChefKissInc/NootedRed#218] [Q&A] [answered]
I do not see NootedRed.kext  file on this repo, how should I get it?

**Comment by truongvantuan** [comment, marked as answer]
@kofany https://github.com/ChefKissInc/NootedRed/actions/runs/7219604635

**Reply by truongvantuan** [reply]
![image](https://github.com/ChefKissInc/NootedRed/assets/22904956/bf8e012c-c1b3-4e8c-917a-61f4e3639b69)
